src/amd64/amd64-64-24k.c

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2017, The Monero Project
```

Comment:
I'm guessing the monero specific code is new ?

Reply:
Changed the copyrights to 2020 in both of these files.